examples/example_patchextraction.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>We can read the image and pass it to the function as a numpy array or instead, you can pass the path of the image to the function.</p><p> </p><p>or instead, you can pass to the function the path to a csv, npy or json file.</p><p> </p><p>please remove variablewindow for now as it needs to be implemented.</p><p> </p><p>On resolution and unit refer to WSIReader.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>When you say refer to WSIReader, do you mean the documentation or the github code? Can you please specify the link here?</p>

Reply:
If you can refer to WSIReader.read_rect() that should be fine